frontend/src/lib/test-shots.ts

```diff
@@ -0,0 +1,432 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import React from 'react'
+import debounce from 'lodash/debounce'
+import capitalize from 'lodash/capitalize'
+import camelCase from 'lodash/camelCase'
+import StackTrace from 'stacktrace-js'
+//import { inspect } from 'util'
+import path from 'path'
+
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
Do we need this in development? What about checking for `test` instead?

Reply:
these are functions meant to help writing tests--to be run in development or test modes but not production